solutions/usaco-688.mdx

```diff
@@ -0,0 +1,58 @@
+---
+id: usaco-688
+source: USACO Bronze 2017 January
+title: Hoof, Paper, Scissors
+author: Ryan Chou
+---
+
+## Solution 1
+[Official Analysis](http://www.usaco.org/current/data/sol_hps_bronze_jan17.html)
```

Comment:
do [Official Analysis (Java)] (http...)

Reply:
i think both of those options are fine (?), but i'll check if there's one we want to standardize

Reply:
i think that the standard should include the language its better that way